### DISCUSSION TITLE
How to stop showing stack icon for converted HEIC in JPG created by PhotoPrism

### DISCUSSION THREAD

**Discussion by noliveleger** [photoprism/photoprism#4040] [Q&A] [answered]
Hello there, 

First of all, I have installed PhotoPrism few days ago to evaluate it (to replace Plex) and congratulations to the devs. It's a really nice product. 

I have some questions I haven't still figured out how to resolve.
The first one is about stacks. Most of my pictures are HEIC. I understand that PhotoPrism needs to convert them to display them.
But each of them shows now the "stack" icon. The HEIC is in my "originals" and the JPG is in "sidebar". 
I would like to keep "Stacks" feature on but to stop having this icon for converted files. Is it possible?

My question may be related to https://github.com/photoprism/photoprism/discussions/1927.

**Comment by graciousgrey** [comment, marked as answer]
Thank you for your kind feedback!

This issue should be fixed with the next release:

- https://github.com/photoprism/photoprism/issues/3993

**Reply by noliveleger** [reply]
Thank you for your quick reply. Never thought to look at issues... Be sure I'll do before asking my other questions ;-)
